$OpenBSD$

Import Unsafe.Coerce for tls-2.1.x VerifyData workaround.
Needed by patch-transport-tls.

--- src/Simplex/Messaging/Transport.hs.orig
+++ src/Simplex/Messaging/Transport.hs
@@ -118,7 +118,8 @@
 import Data.Bifunctor (first)
 import Data.Bitraversable (bimapM)
+import Unsafe.Coerce (unsafeCoerce)
 import Data.ByteString.Char8 (ByteString)
 import qualified Data.ByteString.Char8 as B
 import qualified Data.ByteString.Lazy.Char8 as LB
